cycledash/static/css/examine.css

```diff
@@ -150,14 +150,21 @@ body {
   width: 600px;
   text-align: right;
 }
-.vcf-table .dalliance-open {
+.vcf-table .dalliance-open, .vcf-table .igv-links {
   float: left;
-  background: url(/static/img/magnifying-glass-noun_907.svg) no-repeat 2px 6px;
-  cursor: pointer;
   padding: 3px 0 0 21px;
   line-height: 1.6em;
+}
+.vcf-table .dalliance-open {
+  background: url(/static/img/magnifying-glass-noun_907.svg) no-repeat 2px 6px;
+  cursor: pointer;
   background-size: 15px 15px;
 }
+.vcf-table .igv-links {
+  background: url(/static/img/igv32x32.jpg) no-repeat 2px 3px;
+  background-size: 16px 16px;
```

Comment:
I think 16x16 is a more standard icon size? The `6px` offset vs. the `3px` offset means they don't line up precisely anyway. The different shapes of the two icons make that more about how it looks anyway.